src/forecast_solar/forecast_solar.py

```diff
@@ -85,14 +85,12 @@ async def _request(
         try:
             result = await dns.query("api.forecast.solar", "A")
         except DNSError as err:
-            raise ForecastSolarConnectionError(
-                "Error while resolving Forecast.Solar API address"
-            ) from err
+            msg = "Error while resolving Forecast.Solar API address"
+            raise ForecastSolarConnectionError(msg) from err
```

Comment:
Personally I don't like this  ruff rule. I'd disable it.

Reply:
I like that Ruff applies this rule, but it's a bit of personal preference 😅

Reply:
It's just an additional not needed local variable for the message. 🤷

Reply:
Will put it on the ignore list 👍🏻